corehq/apps/locations/permissions.py

```diff
@@ -208,6 +208,11 @@ def location_safe(view):
     return view
 
 
+# Use this decorator for views that need to be marked location safe but do not actually
+# apply location restrictions to the data they return. e.g. case search
```

Comment:
Should this say something about mobile being the use-case?

Reply:
```suggestion
# apply location restrictions to the data they return e.g. case search. This is generally only applicable to endpoints
# whose client is expected to be the application engine (mobile / web apps).
```

Reply:
:+1: looks good